src/autopas/containers/verletClusterLists/ClusterTower.h

```diff
@@ -396,6 +442,6 @@ class ClusterTower : public ParticleCell<Particle> {
   size_t _numDummyParticles{};
 
   internal::ParticleDeletedObserver *_particleDeletionObserver{nullptr};
-};
+};  // namespace autopas::internal
```

Comment:
Is this supposed to be here? Isn't the following `}` the one with which you exit the namespace?

Reply:
fixed 722ca98fc